src/core/ledger/policies/immediate.js

```diff
@@ -22,16 +22,33 @@ export type ImmediatePolicy = {|
 
 /**
  * Split a grain budget in proportion to the cred scores in
- * the most recent time interval.
+ * the most recent time interval, with the option to extend the interval
+ * to include the last {credIntervalCount} weeks.
  */
 export function immediateReceipts(
   budget: G.Grain,
-  identities: ProcessedIdentities
+  identities: ProcessedIdentities,
+  credIntervalCount: number = 1
```

Comment:
For names, I think `numPeriodsLookback` (or similar) could work as well. I don't object to `credIntervalCount`

Reply:
Used `numPeriodsLookback`.  It might be confusing, since we are now using intervals and periods interchangeably, however, I don't have any better ideas atm so let's roll with that.